tests/stress.rs

```diff
@@ -987,6 +988,7 @@ fn test_selfavatar_config() {
     let mut image_path = PathBuf::from(to_string(blobdir_c));
     image_path.push("delta-logo.png");
 
+    assert!(image_path.as_path().exists());
```

Comment:
The resulting file in the blobdir should exist.

Reply:
It seems windows manages to copy the file to the blobdir at least. This is good to know.